### PR TITLE
gitignore working fine

### PR DESCRIPTION
**Pull request by mahikaw**
#205 gitignore wasn't working before

### CODE REVIEW

**Comment by mahikawason** [comment]
@chhavip Please review. Thanks!

File: PowerUp/.gitignore [hunk 1]
```diff
@@ -9,6 +9,8 @@
 bin/
 gen/
 
+
+
```
Comment (chhavip) [inline PowerUp/.gitignore]: Remove extra line gaps.

**Comment by chhavip** [comment]
So this PR is not having the intended effect of fixing the index of files, trying the changes in my directory led to some unfortunate results. Please explain the approach taken to obtain this PR. This cannot be merged at the point.

**Comment by mahikaw** [comment]
@chhavip, Unfortunately I have deleted this branch and I will have to create a new PR to fix these changes. I will do that asap, if it's fine?

**Comment by chhavip** [comment]
Closing this PR.

**Pull request closed without merging**